### PR TITLE
[Feature] Subscribed to `kytos/topology.links.metadata.(added|removed)` events

### LINKED ISSUE #4: Subscribe to `kytos/topology.links.metadata.(added|removed)`
Currently, pathfinder only [subscribes to topology `kytos.topology.updated` event](https://github.com/kytos-ng/pathfinder/blob/master/main.py#L1980), but topology also publishes [`kytos/topology.links.metadata.(added|removed)` ](https://github.com/kytos-ng/topology/blob/master/main.py#L794) that `pathfinder` is also supposed to subscribe to, just so metadata is also sync'ed as one would expected.

### PR DESCRIPTION
**Pull request by viniarck**
Fixes #4 

### Release notes

`[2.4.0] - 2021-12-20`

#### Added

- Subscribed to to `kytos/topology.links.metadata.(added|removed)` events to update the graph when computing the CSPF accordingly.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [pycodestyle]
 exclude = .eggs,ENV,build,docs/conf.py,venv
+max-line-length = 88
```
Comment (italovalcy) [inline setup.cfg:3]: Just curiosity: this new line length was introduced by black formatting standard?

**Review by italovalcy — APPROVED** [review]
Looks good to me! Pretty neat and important change to reflect changes on link metadata right away to CSPF. Thanks!

File: setup.cfg (hunk 1 above)
Comment (viniarck) [reply setup.cfg:3]: yes, @italovalcy

**Comment by viniarck** [comment]
> Looks good to me! Pretty neat and important change to reflect changes on link metadata right away to CSPF. Thanks!

Thanks for your review, Italo, much appreciated.

**Pull request merged**